### PR TITLE
(CDAP-13033) Fix the start message offset logic when payload table is…

### PR DESCRIPTION
**Pull request by chtyim**
… involved

### CODE REVIEW

**Comment by chtyim** [comment]
Build https://builds.cask.co/browse/CDAP-RUT1479

File: cdap-tms/src/main/java/co/cask/cdap/messaging/service/CoreMessageFetcher.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2016 Cask Data, Inc.
+ * Copyright © 2016-2017 Cask Data, Inc.
```
Comment (anew) [inline cdap-tms/src/main/java/co/cask/cdap/messaging/service/CoreMessageFetcher.java]: it's now 2018 :)

File: cdap-tms/src/test/java/co/cask/cdap/messaging/server/MessagingHttpServiceTest.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2016 Cask Data, Inc.
+ * Copyright © 2016-2017 Cask Data, Inc.
```
Comment (anew) [inline cdap-tms/src/test/java/co/cask/cdap/messaging/server/MessagingHttpServiceTest.java]: ditto

**Review by anew — CHANGES_REQUESTED** [review]
After reading it 3 times, I think it is correct. LGTM After fixing the copyright.

**Comment by chtyim** [comment]
Fixed copyright. Build passed.

**Pull request merged**